src/openrct2-ui/input/KeyboardShortcuts.cpp

```diff
@@ -153,7 +199,17 @@ std::string KeyboardShortcuts::GetShortcutString(int32_t shortcut) const
         format_string(formatBuffer, sizeof(formatBuffer), STR_CMD_PLUS, nullptr);
         String::Append(buffer, sizeof(buffer), formatBuffer);
     }
-    String::Append(buffer, sizeof(buffer), SDL_GetKeyName(SDL_GetKeyFromScancode((SDL_Scancode)(shortcutKey & 0xFF))));
+
+    SDL_Scancode scanCode = static_cast<SDL_Scancode>(shortcutKey & 0xFF);
+    auto keyPair = specialCharNames.find(scanCode);
+    if (keyPair != specialCharNames.end())
+    {
+        format_string(formatBuffer, sizeof(formatBuffer), keyPair->second, nullptr);
+        String::Append(buffer, sizeof(buffer), formatBuffer);
+    }
+    else
+        String::Append(buffer, sizeof(buffer), SDL_GetKeyName(SDL_GetKeyFromScancode(scanCode)));
```

Comment:
Could you put parenthese around this statement? I prefer keeping it consistent within one if-else block.

Reply:
Heh, I prefer the opposite. I'm not a fan of placing unessential braces. As both styles are used in the code base, I'd like to leave it as-is.

Reply:
I think a block should either do
```
if (foo)
    bar();
else
    baz();
```
OR
```
if (foo)
{
    bar();
}
else
{
    baz();
}
```

Not mixing the two. That is just ugly and hard to read.

Reply:
Whether or not it is "just ugly and hard to read" really is a matter of opinion, but not a hill I want this to die on. I have added an amending commit.